tools/config/manage-dependencies.ts

```diff
@@ -22,6 +22,7 @@ import chalk from 'chalk';
 import { execSync } from 'child_process';
 import fs, { readFileSync } from 'fs';
 import glob from 'glob';
+import * as path from 'path';
```

Comment:
This comment is for another file:

Need to bump the angular version in `scripts/install/config.default.sh`